internal/pkg/cli/pipeline_update_test.go

```diff
@@ -4,17 +4,44 @@
 package cli
 
 import (
+	"errors"
+	"fmt"
 	"testing"
 
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/archer"
 	climocks "github.com/aws/amazon-ecs-cli-v2/internal/pkg/cli/mocks"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy"
 	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/manifest"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/term/log"
 	archermocks "github.com/aws/amazon-ecs-cli-v2/mocks"
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	projectName  = "badgoose"
+	region       = "us-west-2"
+	accountID    = "123456789012"
+	pipelineFile = "pipeline.yml"
+	content      = `
```

Comment:
Do you know if these consts are shared across test files? if so it might be worthwhile moving the constants into the test function itself

Reply:
Yeah I am aware of that. I think all the test func can use these constant as default. However, I did not make change to the existing test func, so maybe I can move them to the func.

Reply:
Agh no I changed the existing test func so it is actually using those constants now. And I remembered the reason why I put them out is because to reduce duplication.

Reply:
mm I wonder if this means we can't shadow these variables in other tests. I find it slightly odd that constants used across multiple test files would be inside `pipeline_update_test.go`, it's not an obvious place to look :P

Reply:
OK, then I can move them into the function I created and change the rest back.